ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_math.py

```diff
@@ -2092,3 +2092,43 @@ def test_jax_numpy_fix(
         test_values=False,
         x=x[0],
     )
+
+
+# floor_divide
+@handle_frontend_test(
+    fn_tree="jax.numpy.floor_divide",
+    dtype_values=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric"),
+        num_arrays=2,
+        shared_dtype=True,
+        min_value=2.0,
```

Comment:
We need to test against negative values also this limitation should be removed. Yet you can filter values close to zero to avoid zero division error.